inst/extdata/database/flow_types.csv

```diff
@@ -1,27 +1,358 @@
-flow_id,name,default_unit,energycontent_LHV,energycontent_HHV,density_norm,density_std
-elec,Electricity,MWh,,,,
-heat,Heat,MWh,,,,
-h2,Hydrogen,MWh;LHV,33.33 kWh/kg,39.41 kWh/kg,0.0899 kg/m³,0.0899 kg/m³
-nh3,Ammonia,t,18.6 MJ/kg,22.5 MJ/kg,0.771 kg/m³,0.771 kg/m³
-ch4,Methane,MWh;LHV,13.9 kWh/kg,13.9 kWh/kg,0.716 kg/m³,0.716 kg/m³
-meoh,Methanol,t,5.54 kWh/kg,5.54 kWh/kg,0.791 kg/l,0.791 kg/l
-ng,Natural gas,MWh;LHV,13.1 kWh/kg,13.1 kWh/kg,0.777 kg/m³,0.777 kg/m³
-coal,Coal,MWh;LHV,8 kWh/kg,8 kWh/kg,,
-co2,Purified CO2,t,,,1.977 kg/m³,1.977 kg/m³
-ironore,Iron ore,t,,,,
-dri,Directly reduced iron,t,,,,
-alloys,Alloys,t,,,,
-lime,Lime,t,,,,
-steelliq,Liquid steel,t,,,,
-steelslab,Steel slab,t,,,,
-steelhrc,Hot-rolled coil,t,,,,
-steelscrap,Steel scrap,t,,,,
-hvcs,Higher-value chemicals,t,,,,
-urea,Urea,t,,,1.34 g/cm³,1.34 g/cm³
-water,Water,t,,,1 kg/l,1 kg/l
-oxygen,Oxygen,t,,,0.6998 kg/m³,0.6998 kg/m³
-nitrogen,Nitrogen,t,,,,
-graph_electr,Graphite electrodes,t,,,,
-steam,Steam,MWh,,,,
-ft_fuel,Fischer-Tropsch fuel,MWh;LHV,,,,
-gasoline,Gasoline,MWh;LHV,,,,
+flow_id,attribute,value,source,comment
+elec,name,Electricity,,
+elec,default_unit,MWh,,
+heat,default_unit,MWh,,
+h2,name,Hydrogen,,
+h2,default_unit,MWh;LHV,,
+h2,energycontent_LHV,0.7395 Btu/kg,GREET2008,calculated from values Energy/volume and density given in the source
+h2,energycontent_HHV,0.87465 Btu/kg,GREET2008,calculated from values Energy/volume and density given in the source
+h2,density_norm,0.083752 kg/m³,Lemmon2023,
+h2,density_std,0.089883 kg/m³,Lemmon2023,
+lh2,name,Liquid hydrogen,,
+lh2,default_unit,MWh;LHV,,
+lh2,energycontent_LHV,8174 Btu/kg,GREET2008,calculated from values Energy/volume and density given in the source
+lh2,energycontent_HHV,9653.36 Btu/kg,GREET2008,calculated from values Energy/volume and density given in the source
```

Comment:
Is it really correct that this value for liquid hydrogen is pretty much by a factor 10³ greater than its value for gaseous hydrogen (a few lines above)?

Reply:
Thanks for the remark, this is obviously not correct, but the result of a unit conversion error, which also affected some other values. Have fixed it in the most recent commit